### DISCUSSION TITLE
[#995500] Variants not being shown in block picker/plank blocks not being shown in fill selection

### DISCUSSION THREAD

**Discussion by megatron0001** [Mojang/minecraft-editor#83] [Bugs]
Birch, dark oak, jungle, acacia, spruce planks when enter in fill selection area does not show or work.
<img width="345" alt="image" src="https://user-images.githubusercontent.com/82678869/224616869-f2111ed6-23a6-491b-81d4-25603d95fa53.png">
<img width="324" alt="image" src="https://user-images.githubusercontent.com/82678869/224616892-8d022edc-a75d-4c4c-a990-2b5933e69e1b.png">
<img width="336" alt="image" src="https://user-images.githubusercontent.com/82678869/224616925-94bea870-6bd6-4ce3-ad0c-3ff25bafe27c.png">
<img width="333" alt="image" src="https://user-images.githubusercontent.com/82678869/224616961-e6dda7b0-f6fe-442b-8496-af68b96d3725.png">

**Comment by ThatJadon26** [comment]
That's because they all have the identifier `planks`

**Reply by megatron0001** [reply]
How would I get birch planks... :(

**Comment by minecraftchloe** [comment]
Ah! Yes. Right now we're pulling from a blocks API list, which happens to not include all of the variants. We have some major improvements to make towards a block palette that includes _everything_. 

So...expected behavior _for now_ 😭 and we have known work to do to surface everything in the list 🎉

(One workaround I've been using for myself is to place what I want in Crosshair, and then use select -> copy/paste.)
